db/seeds.rb

```diff
@@ -1,9 +1,98 @@
-# This file should ensure the existence of records required to run the application in every environment (production,
-# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Example:
-#
-#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
```

Comment:
If you're updating this file this can go too ;) but not important

Reply:
cool, cool. I already deleted those lines. I think you see them here highlighted in red just as an fyi that they are now gone/changed.

The first line of my branched seed file starts at line 1 that you see here in green (..where I ask if we want to delete or not)

Reply:
Awesome, that makes a lot of sense, thanks!